src/parser.rs

```diff
@@ -1989,6 +1993,7 @@ mod tests {
         Identifier,
         ParenL,
         ParenR,
+        Slash,
```

Comment:
I wasn't able to confidently understand what these tests mean. My best guess was that these are the tokens that can be found at the beginning of an expression without causing an exception.

This was the output from one of the two failed tests:
```
---- parser::tests::unclosed_parenthesis_in_interpolation stdout ----
thread 'parser::tests::unclosed_parenthesis_in_interpolation' panicked at 'assertion failed: `(left == right)`

Diff < left / right > :
 CompileError {
     token: Token {
         offset: 15,
         length: 2,
         line: 1,
         column: 12,
         src: "a:\n echo {{foo(}}",
         kind: InterpolationEnd,
     },
     kind: UnexpectedToken {
         expected: [
             Backtick,
             Identifier,
             ParenL,
             ParenR,
<            Slash,
             StringToken,
         ],
         found: InterpolationEnd,
     },
 }

', src/parser.rs:907:9
```

Reply:
All that said, I am wondering if my understanding was correct and whether adding `Slash` to the list was the appropriates change.

Reply:
Yup, I think that's correct. When the parser hits an unexpected token, the error message includes which tokens might have been accepted. Since `/` can now appear at the beginning of an expression, the error message now includes `/`, so adding it to the `expected` token list is correct.